### PR TITLE
Disallow selecting ClusterHostSystem in compute resource selector widget

### LINKED ISSUE #256: VCH Creation Wizard allows for selection of a host within a cluster
**VIC version:**

1.3.0 OVA RC 4 (UI RC 6)

**Steps to reproduce:**

Launch the creation wizard, select a host in a cluster.

**Actual behavior:**

Creation fails with an error like `Failed to validate VCH: compute resource '/dc1/host/cluster1/...' not found`

**Expected behavior:**

Hosts within a cluster are visible, but not selectable as compute resources. (Stand-alone hosts, not in a cluster, should be both visible and selectable.)

**Additional information:**

This limitation may change in the future, but is captured in [the docs](https://vmware.github.io/vic-product/assets/files/html/1.2/vic_vsphere_admin/vch_placement.html#-compute-resource-):

> **NOTE:** You cannot deploy a VCH to a specific host in a cluster. You deploy the VCH to the cluster, and DRS manages the placement of the VCH on a host.

### PR DESCRIPTION
**Pull request by jooskim**
Fixes #256

### CODE REVIEW

File: h5c/vic/src/vic-webapp/e2e/vch-create-wizard/1-basic.e2e-spec.ts [hunk 1]
```diff
@@ -198,18 +198,12 @@ describe('VCH Create Wizard - Basic', () => {
...
-      element.isPresent().then(present => {
-        if (present) {
-          element.getText().then(function(text) {
-            if (deletedVch.test(text)) {
-              vchFound = true;
-            }
-          });
-        }
-      })
+    const vchClrDgActionXpath = `//clr-dg-action-overflow[contains(@class, '${namePrefix + specRunId}')]`;
```
Comment (jak-atx) [inline h5c/vic/src/vic-webapp/e2e/vch-create-wizard/1-basic.e2e-spec.ts]: I'm seeing this timeout 
✓ should delete created vch
//clr-dg-action-overflow[contains(@class, 'virtual-container-host-866')] true
: : 
✗ should verify the created vch has been deleted
      - Failed: Wait timed out after 60092ms

Comment (jak-atx) [reply h5c/vic/src/vic-webapp/e2e/vch-create-wizard/1-basic.e2e-spec.ts]: Also when it tries to discover if the vch is powered on (similar query) it fails with...

StaleElementReferenceError: stale element reference: element is not attached to the page document
  (Session info: chrome=63.0.3239.132)
  (Driver info: chromedriver=2.35.528157 (4429ca2590d6988c0745c24c8858745aaaec01ef),platform=Mac OS X 10.11.6 x86_64) not found

**Comment by jooskim** [comment]
Verified the tests ran successfully on a local run.

**Comment by jooskim** [comment]
```
 VCH Create Wizard - Basic
    ✓ should redirect to login
    ✓ should login
    ✓ should navigate to vsphere home
    ✓ should navigate to vic plugin
    ✓ should navigate to summary tab
    ✓ should navigate to vch tab
    ✓ should open create vch wizard
    ✓ should input vch name
    ✓ should complete general step
    ✓ should select a compute resource
    ✓ should complete compute capacity step
[15:13:40] W/element - more than one element found for locator by.cssContainingText("select#image-store-selector option", "datastore1") - the first result will be used
    ✓ should select a datastore
    ✓ should complete storage capacity step
    ✓ should select a bridge network
[15:13:42] W/element - more than one element found for locator by.cssContainingText("select#public-network-selector option", "network") - the first result will be used
    ✓ should select a public network
    ✓ should complete networks step
    ✓ should complete security step
    ✓ should complete registry access step
    ✓ should enter ops user creds
    ✓ should complete ops user step
    ✓ should create a vch
.pagination-next not found
    ✓ should find the new vch in datagrid
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
Power On virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:07 PM
StaleElementReferenceError: stale element reference: element is not attached to the page document
  (Session info: chrome=63.0.3239.132)
  (Driver info: chromedriver=2.35.528157 (4429ca2590d6988c0745c24c8858745aaaec01ef),platform=Mac OS X 10.12.6 x86_64) not found
Reconfigure virtual machine: virtual-container-host-1085: 01/18/2018, 3:14:37 PM
    ✓ should verify the new vch has properly started
    ✓ should delete created vch
//clr-dg-action-overflow[contains(@class, 'virtual-container-host-1085')] true
Power Off virtual machine: virtual-container-host-1085:
Power Off virtual machine: virtual-container-host-1085:
Power Off virtual machine: virtual-container-host-1085:
Power Off virtual machine: virtual-container-host-1085:
Power Off virtual machine: virtual-container-host-1085:
Power Off virtual machine: virtual-container-host-1085:
Power Off virtual machine: virtual-container-host-1085:
Power Off virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Reconfigure virtual machine: virtual-container-host-1085:
Delete file: datastore1: 01/18/2018, 3:14:54 PM
Delete file: datastore1: 01/18/2018, 3:14:54 PM
Delete file: datastore1: 01/18/2018, 3:14:54 PM
Delete file: datastore1: 01/18/2018, 3:14:54 PM
Delete file: datastore1: 01/18/2018, 3:14:54 PM
Delete file: datastore1: 01/18/2018, 3:14:54 PM
Delete resource pool: virtual-container-host-1085: 01/18/2018, 3:14:55 PM
    ✓ should verify the created vch has been deleted

Executed 25 of 25 specs SUCCESS in 1 min 51 secs.
[15:14:56] I/launcher - 0 instance(s) of WebDriver still running
[15:14:56] I/launcher - chrome #01 passed
```

**Pull request merged**